python-sdk/tests_integration/sql/operators/transform/test_transform.py

```diff
@@ -249,6 +406,32 @@ def select(input_table: Table):
     test_utils.run_dag(sample_dag)
 
 
+def test_transform_using_table_metadata_mssql(sample_dag):
```

Comment:
do we need to add `@pytest.mark.integration`?

Reply:
fixed